### ISSUE TITLE
NVDA screen reader doesn't read slides' content on Firefox

### ISSUE THREAD

**Issue by wujekbogdan** [nolimits4web/swiper#3436]
This is a:
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: 5.3.0 (happens also on older versions)
* Platform/Target and Browser Versions:  Firefox 71.0 on Windows (happens on older versions of FF too)
* Live Link or JSFiddle/Codepen or website with isssue: It could be tested on this official demo https://swiperjs.com/demos/020-navigation.html

### What you did
I was testing the Swiper accessibility on various browsers with [NVDA screen reader](https://www.nvaccess.org/). It works well everywhere except for Firefox.

By default, the NVDA reader reads all texts on hover. It works for all texts except for texts that are inside slides. I found that the issue is caused by inline styles for the `.swiper-wrapper` element. The problem is caused by the `transform` property. If I disable that property then texts are read by NVDA (but the slider breaks, of course).

What's interesting is that NVDA is able to read`alt` attributes if there are images within slides.

### Expected Behavior
NVDA screen reader should read slides' content on hover.

### Actual Behavior
NVDA screen reader doesn't read slides' content on hover.
